### PR TITLE
[i18nIgnore][prefetch.mdx]: Adds a `load` prefetch strategy

### PR DESCRIPTION
**Pull request by wtto00**
#### Description (required)

Adds a `'load'` prefetch strategy to prefetch links on page load

#### Related issues & labels (optional)

- Closes #
- Suggested label: 

#### For Astro version: `4.x`. See astro PR [#9513](https://github.com/withastro/astro/pull/9513).

### CODE REVIEW

**Review by sarah11918 — APPROVED** [review]
English contribution is approved! Trying an experiment to see if we can get all the other language reviews in this PR, too!

File: src/content/docs/zh-cn/guides/prefetch.mdx [hunk 1]
```diff
@@ -32,11 +32,12 @@ export default defineConfig({
 
 ### 预获取策略
 
-Astro 支持 3 种预获取策略，以适应各种用例：
+Astro 支持 4 种预获取策略，以适应各种用例：
 
 - `hover`（默认）：当你将鼠标悬停在链接上或聚焦在链接上时预获取。
 - `tap`：在你点击链接之前预获取。
 - `viewport`：当链接进入视口时预获取。
+- `load`: 当页面加载后预获取当前页面的所有链接。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/prefetch.mdx:40]:
```suggestion
- `load`：当页面加载后预获取当前页面的所有链接。
```

**Comment by liruifengv** [comment]
Just a punctuation mark. Everything else is great in zh-cn.

**Review by liruifengv — APPROVED** [review]

File: src/content/docs/en/guides/prefetch.mdx [hunk 2]
```diff
@@ -32,11 +32,12 @@ The `prefetch` config also accepts an option object to further customize prefetc
 
 ### Prefetch strategies
 
-Astro supports 3 prefetch strategies for various use cases:
+Astro supports 4 prefetch strategies for various use cases:
 
 - `hover` (default): Prefetch when you hover over or focus on the link.
 - `tap`: Prefetch just before you click on the link.
 - `viewport`: Prefetch as the links enter the viewport.
+- `load`: Prefetch all links on the page after page is loaded.
```
Comment (Yan-Thomas) [inline src/content/docs/en/guides/prefetch.mdx:40]: Small nit (shouldn't affect translations)
```suggestion
- `load`: Prefetch all links on the page after the page is loaded.
```

File: src/content/docs/es/guides/prefetch.mdx [hunk 3]
```diff
@@ -32,11 +32,12 @@ La configuración `prefetch` también acepta un objeto de opciones para personal
 
 ### Estrategias de precarga
 
-Astro admite 3 estrategias de precarga para varios casos de uso:
+Astro admite 4 estrategias de precarga para varios casos de uso:
 
 - `hover` (por defecto): Precarga cuando pasas el cursor o te enfocas en el enlace.
 - `tap`: Precarga justo antes de hacer clic en el enlace.
 - `viewport`: Precarga cuando los enlaces entran en el viewport.
+- `load`: Precargar todos los enlaces de la página después de que la página se cargue.
```
Comment (Yan-Thomas) [inline src/content/docs/es/guides/prefetch.mdx:40]: Not native Spanish speaker, but pretty sure the verb tense here should be this one instead.
```suggestion
- `load`: Precarga todos los enlaces de la página después de que la página se cargue.
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks @wtto00, left a few couple nits.

File: src/content/docs/es/guides/prefetch.mdx [hunk 4]
```diff
@@ -32,11 +32,12 @@ La configuración `prefetch` también acepta un objeto de opciones para personal
 
 ### Estrategias de precarga
 
-Astro admite 3 estrategias de precarga para varios casos de uso:
+Astro admite 4 estrategias de precarga para varios casos de uso:
 
 - `hover` (por defecto): Precarga cuando pasas el cursor o te enfocas en el enlace.
 - `tap`: Precarga justo antes de hacer clic en el enlace.
 - `viewport`: Precarga cuando los enlaces entran en el viewport.
+- `load`: Precarga todos los enlaces de la página después de que la página se cargue.
```
Comment (Waxer59) [inline src/content/docs/es/guides/prefetch.mdx:40]:
```suggestion
- `load`: Precarga todos los enlaces en la página después de que la página se ha cargado.
```

**Review by Waxer59 — COMMENTED** [review]
Just a suggestion on the translation of `es`.

**Pull request merged**